chainercv/links/loss/semantic_segmentation_loss.py

```diff
@@ -0,0 +1,96 @@
+import chainer
+from chainer import cuda
+import chainer.functions as F
+from chainer import reporter
+
+import numpy as np
+
+from chainercv.evaluations import eval_semantic_segmentation
+
+
+def segmentation_accuracies(y, t, n_class):
+    y = F.argmax(F.softmax(y), axis=1)
+    if y.ndim == 3:
+        y = y[:, None, :, :]
+    if t.ndim == 3:
+        t = t[:, None, :, :]
+    return [np.mean(ret)
+            for ret in eval_semantic_segmentation(y.data, t.data, n_class)]
+
+
+class PixelwiseSigmoidLoss(chainer.Chain):
+
+    def __init__(self, model, n_class, calc_accuracy=True):
+        super(PixelwiseSigmoidLoss, self).__init__(predictor=model)
+        self.n_class = n_class
+        self.calc_accuracy = calc_accuracy
+
+    def __call__(self, x, t):
+        self.y = self.predictor(x)
+        self.loss = F.sigmoid_cross_entropy(self.y, t)
+        reporter.report({'loss': self.loss}, self)
+        if self.calc_accuracy:
```

Comment:
How about `compute_accuracy` instead of `calc_accuracy`.
This is the name used in Chainer's links/model/classifier.py